mods/ra/rules/campaign-rules.yaml

```diff
@@ -60,7 +60,7 @@ MONEYCRATE:
 	GiveCashCrateAction:
 		Amount: 2000
 	RenderSprites:
-		Image: scrate
+		Image: wcrate
```

Comment:
`wcrate` is already the default defined in rules/misc.yaml, so we can instead just delete this override.

It turns out that wcrate.shp is actually using player remap, and by eye it looks like it's using France's colour range. We can therefore fix this by changing the definition in rules/misc.yaml to:

```
RenderSprites:
	Image: wcrate
	Palette: wcrate
```

and adding a new palette definition in rules/palettes.yaml:
```
IndexedPalette@wcrate:
	Name: wcrate
	BasePalette: player
	Index: 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95
	ReplaceIndex: 224, 224, 225, 225, 226, 184, 185, 186, 187, 188, 188, 189, 190, 190, 191, 191
```

Reply:
Thanks! I'll tackle that in a separate PR.